examples/tut_add-vectors.cpp

```diff
@@ -166,36 +184,38 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 #if defined(RAJA_ENABLE_HIP)
   std::cout << "\n Running RAJA HIP vector addition...\n";
 
-  int *d_a = memoryManager::allocate_gpu<int>(N);
-  int *d_b = memoryManager::allocate_gpu<int>(N);
-  int *d_c = memoryManager::allocate_gpu<int>(N);
+  Resource hipRc{Cuda()};
```

Comment:
We need to add a HIP resource?

Reply:
We have one, is it not sufficient? https://github.com/LLNL/camp/blob/master/include/camp/resource/hip.hpp

Reply:
Oops, copy paste error. Just pushed a fix.